awd/cli.py

```diff
@@ -27,159 +27,81 @@
 
 
 @click.group()
-@click.option(
-    "--aws_region",
-    required=True,
-    default=config.AWS_REGION_NAME,
-    help="The AWS region to use for clients.",
-)
-@click.option(
-    "--log_level",
-    required=True,
-    default=config.LOG_LEVEL,
-    help="The log level to use.",
-)
-@click.option(
-    "--doi_table_name",
-    required=True,
-    default=config.DOI_TABLE,
-    help="The DynamoDB table containing the state of DOIs in the workflow.",
-)
-@click.option(
-    "--sqs_base_url",
-    required=True,
-    default=config.SQS_BASE_URL,
-    help="The base URL of the SQS queues.",
-)
-@click.option(
-    "--sqs_output_queue",
-    required=True,
-    default=config.SQS_OUTPUT_QUEUE,
-    help="The SQS queue from which results messages will be received.",
-)
-@click.option(
-    "--log_source_email",
-    required=True,
-    default=config.LOG_SOURCE_EMAIL,
-    help="The email address sending the logs.",
-)
-@click.option(
-    "--log_recipient_email",
-    required=True,
-    default=config.LOG_RECIPIENT_EMAIL,
-    help="The email address receiving the logs.",
-)
 @click.pass_context
 def cli(
     ctx: click.Context,
-    aws_region: str,
-    log_level: str,
-    doi_table_name: str,
-    sqs_base_url: str,
-    sqs_output_queue: str,
-    log_source_email: str,
-    log_recipient_email: str,
 ) -> None:
-    sentry_dsn = config.SENTRY_DSN
+    config = Config()
+    sentry_dsn = config.SENTRY_DSN  # type: ignore[attr-defined]
```

Comment:
If the approach outlined below around the `Config` class is taken, where all the env vars are included as annotations (e.g. `WORKSPACE: str`), then these linting skips won't be needed.